src/Components/Components/src/ComponentBase.cs

```diff
@@ -177,188 +177,93 @@ void IComponent.Configure(RenderHandle renderHandle)
         public virtual Task SetParametersAsync(ParameterCollection parameters)
```

Comment:
I need to look at this in more detail, I'm not clear if more renders are being introduced.

Reply:
@javiercn Please let us know if you have had a chance to do this.